helpdesk-bot/src/test/java/org/symphonyoss/symphony/bots/helpdesk/bot/it/TestContext.java

```diff
@@ -0,0 +1,51 @@
+package org.symphonyoss.symphony.bots.helpdesk.bot.it;
+
+import org.symphonyoss.client.model.Room;
+import org.symphonyoss.symphony.clients.model.SymUser;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Created by crepache on 22/02/18.
+ * Class to store the variables to use on any part of tests.
+ */
+public class TestContext {
+
+  private static TestContext instance;
+
+  private Map<UsersEnum, SymUser> users = new HashMap();
+
+  private Room queueRoom;
+
+  private TestContext() {
+  }
+
+  public static synchronized TestContext getInstance() {
```

Comment:
You might remove this synchronized directive if you perform the eager initialization. You should remove the null check as well.

Reply:
Done.